update R version to 4.2.2

vandenman opened the pull request:
Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2146

merge https://github.com/jasp-stats/jasp-actions/pull/17 at the same time

the new versions are uploaded at https://static.jasp-stats.org/development/{version}

vandenman committed:
update R version to 4.2.2

JorisGoosen commented:
Hmm, the PR seems right but the R_VERSION is cached and therefore qt keeps downloading R.4.2.1 instead of the newer one.
After changing the cmake-var manually, throwing away the Frameworks folder and retrying I get the new one.
But Ill see if I can just turn off the caching

JorisGoosen committed:
I guess we shouldnt cache the R_VERSION as it then doesnt want to upgrade.

Now Im not sure how we would support going back to an older version, it wont be simply changing a cmake settings I suppose.
But the way forward is where we want to go

JorisGoosen reviewed (approved):
now it doesnt cache the version anymore, although to use the new version of R the framework ought to be thrown away anyway
